libclc/generic/lib/integer/ctz.cl

```diff
@@ -0,0 +1,42 @@
+#include "../clcmacro.h"
+#include <clc/clc.h>
+
+_CLC_OVERLOAD _CLC_DEF short ctz(short x) {
+  return x ? __builtin_ctzs(x) : 16;
+}
+
+_CLC_OVERLOAD _CLC_DEF ushort ctz(ushort x) {
+  return x ? __builtin_ctzs(x) : 16;
+}
+
+_CLC_OVERLOAD _CLC_DEF char ctz(char x) {
```

Comment:
Consistent with llvm way. It is recommended to adjust the situation of char and uchar in the front.